src/utils.py

```diff
@@ -28,10 +28,14 @@ class HPOSupportedTypes:
     N_JOBS = 1
     N_TRIALS = 10
     SERVER_HOSTNAME = "0.0.0.0"
-    SERVER_PORT = 8085
+    #SERVER_PORT = 8085
```

Comment:
Please remove this code if this is not going to be needed

Reply:
done.